grakn/client.py

```diff
@@ -44,8 +44,8 @@ def core(address=DEFAULT_ADDRESS) -> "GraknClient":
         return _RPCGraknClient(address)
 
     @staticmethod
-    def cluster(address=DEFAULT_ADDRESS) -> "GraknClient":
-        return _RPCGraknClientCluster(address)
+    def cluster(addresses: List[str]) -> "GraknClient":
+        return _RPCGraknClientCluster(addresses)
```

Comment:
This change drops support for constructing `GraknClient.cluster()` with no parameters; but we're fine with that - the default address is pretty nonsensical for Cluster. Developers won't be running Cluster on their machines.